### DISCUSSION TITLE
GitHub profile privacy

### DISCUSSION THREAD

**Discussion by luislortega** [community/community#42148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be very interesting if the github profiles could be made totally private... I mean if you don't want to be found by search engines, put "don't be found by search engines" or if you totally don't want anyone to see anything anymore about your profile... well, putting "private profile" is an idea... I feel that there are details to refine on this, but without doubts it is something that could greatly improve the user experience... personally I would like make my profile private, for example all the repositories or my activity history but still being able to use the repositories and that people can see my activity.

**Comment by vojkny** [comment]
Yes plaese, also noted here: https://github.com/orgs/community/discussions/31372
